DJHelper/DJHelper/View Controllers/DJLoginViewController.swift

```diff
@@ -43,7 +43,7 @@ class DJLoginViewController: UIViewController, UITextFieldDelegate {
         // if username is found, set the currentHost variable to the Host fetched from the fetchRequest.
         // if not found, present an alert and prompt to the registration scene.
         let fetchRequest: NSFetchRequest<Host> = Host.fetchRequest()
-        let predicate = NSPredicate(format: "\(#keyPath(Host.username)) == \(username)")
+        let predicate = NSPredicate(format: "username == %@", username)
```

Comment:
Oh. Why did you choose to not use #KeyPath for configuring the predicate?

Reply:
It didn't work at first so I had to problem solve. In the end, I was more familiar with the format I used so that's what I did.  The keypath probably would work if I removed the "Host." from the path, but I was just happy to make it work.